plugins/pastefromword/filter/default.js

```diff
@@ -205,7 +205,15 @@
 						delete element.attributes.size;
 					}
 
-					createAttributeStack( element, filter );
+					// Create style stack for td/th > font if only class
+					// and style attributes are present. Such markup is produced by Excel.
+					if ( CKEDITOR.dtd.tr[ element.parent.name ] &&
+						CKEDITOR.tools.arrayCompare( CKEDITOR.tools.objectKeys( element.attributes ), [ 'class', 'style' ] ) ) {
+
+						Style.createStyleStack( element, filter, editor );
```

Comment:
Since you reverted `topmostElementName` usage here, it's no longer needed and can be removed from `createStyleStack` method.